frontend/src/app/render_wasm/api.cljs

```diff
@@ -106,8 +106,22 @@
   [clip-content]
   (h/call internal-module "_set_shape_clip_content" clip-content))
 
+(defn- translate-shape-type
+  [type]
+  (case type
```

Comment:
Probably we should include some doc about this in  render-wasm/docs/serialization.md